src/TesApi.Web/BatchScheduler.cs

```diff
@@ -602,7 +602,9 @@ private async Task<CloudTask> ConvertTesTaskToBatchTaskAsync(TesTask task, bool
             
             var filesToDownload = await Task.WhenAll(
                 inputFiles
-                .Where(f => f?.Url?.StartsWith("drs://", StringComparison.OrdinalIgnoreCase) != true) // do not attempt to download DRS input files since the cromwell-drs-localizer will
+                .Where(
+                    f => f?.Url?.StartsWith("drs://", StringComparison.OrdinalIgnoreCase) != true  // do not attempt to download DRS input files since the cromwell-drs-localizer will
+                    && f?.Streamable != true) // Don't download files where localization_optional is set to true in the WDL (corresponds to "Streamable" property being false on TesInput)
```

Comment:
Is this comment accurate? If localization_optional is set to true, then Streamable property is false? Shouldn't Streamable be true?